package.json

```diff
@@ -8,7 +8,8 @@
     "test": "karma start",
     "prepublish": "typings install && webpack && webpack --config webpack.vendor.config.js",
     "build-docs": "jsdoc -c jsdoc-conf.json",
-    "bower": "bower install"
+    "bower": "bower install",
+    "postinstall": "bower install"
```

Comment:
@karelhala is the `bower` npm target  ("bower": "bower install") even used?
Can we get rid of it?